debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/Scn.java

```diff
@@ -15,29 +15,91 @@
  */
 public class Scn implements Comparable<Scn> {
 
+    /**
+     * Represents an Scn that is considered INVALID, useful as a placeholder.
+     */
     public static final Scn INVALID = new Scn(new BigDecimal(-1));
+
+    /**
+     * Represents an Scn that implies the maximum possible value of an SCN, useful as a placeholder.
+     */
+    public static final Scn MAX = new Scn(new BigDecimal(-2));
+
+    /**
+     * Represents an Scn with a value of {@code 0}.
+     */
     public static final Scn ZERO = new Scn(BigDecimal.ZERO);
-    public static final Scn ONE = new Scn(BigDecimal.ONE);
 
-    private BigDecimal scn;
+    private final BigDecimal scn;
```

Comment:
Should this be `BigInteger` actually? I.e. do we really expect fractional values here?

Reply:
Yes, there is an existing Jira to make this a `BigInteger` that's part of 1.6 iirc.

Reply:
Ok, yeah, that'd be good to address.